Sources/Actions/Profile/ShowPosts.php

```diff
@@ -859,6 +860,12 @@ protected function loadPosts(bool $is_topics = false): void
 			Lang::censorText($row['body']);
 			Lang::censorText($row['subject']);
 
+			// Old SMF versions autolinked during output rather than input,
+			// so maintain expected behaviour for those old messages.
+			if (version_compare($this->version, '3.0', '<')) {
```

Comment:
I don't see this property defined at a very cursory glance. And it  looks like this isn't the only one that looks out of place.

Reply:
Oof. That should be `$row['version']`.

Reply:
Fixed.